packages/wallets/src/features/cfd/modals/DxtradeEnterPasswordModal/DxtradeEnterPasswordModal.tsx

```diff
@@ -162,7 +162,7 @@ const DxtradeEnterPasswordModal = () => {
         if (!isSuccess && accountStatusSuccess) {
             return isDxtradePasswordNotSet ? (
                 <CreatePassword
-                    icon={<DxTradePasswordIcon />}
+                    Icon={<DxTradePasswordIcon />}
```

Comment:
why did you change it?

Reply:
Gitguardian giving me this:
![image](https://github.com/binary-com/deriv-app/assets/103104395/224a0c4b-6137-4c8c-8054-484ef5ce9ceb)
So I decided to change the name and see if there's a difference.